dump_utils.hpp

```diff
@@ -10,7 +10,6 @@
 #include <phosphor-logging/elog-errors.hpp>
 #include <phosphor-logging/elog.hpp>
 #include <phosphor-logging/log.hpp>
-#include <sdbusplus/bus.hpp>
```

Comment:
why is this removed not related to your change

Reply:
Actually I realized that it was an unnecessary inclusion thus removed. But re-included on your suggestion